light-zk.js/src/test-utils/userTestAssertHelper.ts

```diff
@@ -697,8 +697,8 @@ export class UserTestAssertHelper {
     // relayer recipient's sol balance should be increased by the relayer fee
     await this.assertSolBalance(
       this.tokenCtx!.symbol != "SOL" &&
-        !this.recipient.preTokenBalance &&
-        this.testInputs.type == Action.UNSHIELD
+        // !this.recipient.preTokenBalance &&
```

Comment:
can this be removed or why is it commented?

Reply:
it was breaking the test I will remove it

Reply:
done